scripts/ci-e2e.sh

```diff
@@ -36,8 +36,8 @@ WORKING_DIR=/opt/metal3-dev-env
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
 
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=main
+M3_DEV_ENV_REPO="https://github.com/Nordix/metal3-dev-env.git"
+M3_DEV_ENV_BRANCH=ci_fix_e2e_v1a5/adil
```

Comment:
Let's revert this and once https://github.com/metal3-io/metal3-dev-env/pull/1036 is merged, re-run e2e tests here.